packages/middleware/src/types/config.ts

```diff
@@ -16,12 +16,15 @@ export type Integrations<TIntegrationsContext extends TObject = TObject> = {
     : never;
 };
 
+export type LoggerConfig = unknown;
```

Comment:
Is this a mistake? We know the config interface: 
```
export const loggerConfig = {
  level: LogLevel.Error, //optional
    includeStackTrace: true, //optional,
    [key: string]: any, //optional
      handler: { // optional
        info: ...
        warn: ....
        // and so on
      }
    }
};
```

Reply:
I see that `ConsolaLoggerOptions` is not exported, only `LoggerOptions` is available. Should I use the latter then?

Reply:
You should write it with your own hands, We do not want to leak internals of subpackages to the public user interface.

Reply:
It is not exported anymore as I just use 
```ts
import {
  type LoggerOptions,
} from "@vue-storefront/logger";
```

I am not reexporting this type outside of the package. Would you still want me to copy paste it to the package? (LoggerOptions)